stacks/abax-minuba/Pulumi.prod.yaml

```diff
@@ -0,0 +1,18 @@
+config:
+  abax:client-id:
+    secure: AAABAKk4157pwBBVsoW9bub4gHC/jeS/+OZ8VLMgGxz2W6aRb5RVbg/fIvu4HtnODlGKJ9PP+Vqc6iyTivtIyg==
+  abax:client-secret:
+    secure: AAABAF/kuvtrSu2dw6axE4N1MK3KaaOC3WEYbP4+dQGJ1SZ48RPH+bUb5/53GO8ceRH4JQKcSBLGIniuGNuhlw==
```

Comment:
I think we should move most of these configuration values to a Pulumi Environment, especially the secrets.